src/libcadet/model/parts/TwoDimensionalConvectionDispersionOperator.cpp

```diff
@@ -872,30 +874,22 @@ bool TwoDimensionalConvectionDispersionOperator::configure(UnitOpIdx unitOpIdx,
 bool TwoDimensionalConvectionDispersionOperator::notifyDiscontinuousSectionTransition(double t, unsigned int secIdx)
 {
 	bool hasChanged = false;
+	std::vector<int> dir_old = _dir;
```

Comment:
I think we can avoid this local (and memory allocating) variable. See below for a suggestion.